andela/casablanca-rc
# Ft actionable analytics

## What does this PR do?
This PR implements a feature of actionable analytics for sellers and owners within the application.
 
## Description of Task to be completed?
- add analytics to register package
- add data and inventory
- add product statistics tab
- add side nav with options 
- add dashboard box display
 - align colour scheme to team defined colours.

## How should this be manually tested?
* clone the repo and change into the project directory
* run npm install to install required packages
* create a .env file and add all the required environment variables
* start up the server with the command reaction
* visit http://localhost:3000 and to load application in your browser
* sign in as a seller or admin
* click on analytics to see analytics dashboard

## Any background context you want to provide?
<img width="1013" alt="screen shot 2018-05-24 at 9 42 05 am" src="https://user-images.githubusercontent.com/8572766/40474826-ec169a28-5f37-11e8-9efd-0282dc92796a.png">

<img width="1081" alt="screen shot 2018-05-24 at 9 42 34 am" src="https://user-images.githubusercontent.com/8572766/40474862-fa99e686-5f37-11e8-80c9-9deffc7d3e28.png">



## What are the relevant pivotal tracker stories?(if applicable)
 #157032084
